openedx/features/enterprise_support/api.py

```diff
@@ -473,16 +473,18 @@ def consent_needed_for_course(request, user, course_id, enrollment_exists=False)
     if not enterprise_learner_details:
         consent_needed = False
     else:
+        currently_active_enterprise = enterprise_customer_uuid_for_request(request)
```

Comment:
We should have a test to support this change. e.g. consent_needed is False for ORAP and True otherwise